src/models/HybridSystem.jl

```diff
@@ -0,0 +1,172 @@
+"""
+Representation of a Hybrid System that collects renewable generation, load, thermal generation
+and storage
+"""
+mutable struct HybridSystem{
+    T <: ThermalGen,
+    L <: ElectricLoad,
+    S <: Storage,
+    R <: RenewableGen,
+} <: StaticInjection
+    name::String
+    available::Bool
+    status::Bool
+    bus::Bus
+    active_power::Float64
+    reactive_power::Float64
+    thermal_unit::T
+    electric_load::L
+    storage::S
+    renewable_unit::R
+    # PCC Data
+    "Thermal limited MVA Power Output of the unit. <= Capacity"
+    pcc_impedance::ComplexF64
+    rating::Float64
```

Comment:
A question here: so is this `rating` parameter for each component rating/size, or also for inverter rating? I'm just wondering which parameters specify the e.g ILR of the inverter.

Reply:
Rating works as a proxy to limit the interconnection's max current. This can be implemented as 
```P^2 + Q^2 <= rating^2``` or ```V*I < rating``` depending on the modeling needs.